.github/workflows/pipeline.yml

```diff
@@ -58,11 +58,18 @@ jobs:
           path: ${{ steps.print-esy-cache.outputs.esy-cache }}
           key: ${{ matrix.os }}-esy-${{ hashFiles('**/index.json') }}
 
-      - name: Build ppx
+      - name: Build ppx (non-linux)
+        if: matrix.os != 'ubuntu-latest'
         run: |
           cd lib
           esy build
 
+      - name: Build ppx (linux)
+        if: matrix.os == 'ubuntu-latest'
+        run: |
+          cd lib
+          esy dune build -p '#{self.name}' --profile release-static
```

Comment:
I couldn't get this to work as `esy linux-release-static`, I'm not sure why - it was added to the package.json as described here: https://esy.sh/docs/en/configuration.html#project-specific-commands

The command also fails on my ubuntu laptop, so I'm not sure what is wrong.
![image](https://user-images.githubusercontent.com/6032276/110204915-44c9ff80-7e7e-11eb-8b1e-47ccf2d7cb99.png)